src/main/converter/publication.ts

```diff
@@ -61,9 +63,29 @@ export class PublicationViewConverter {
             lcp: document.lcp,
             lcpRightsCopies: document.lcpRightsCopies,
 
-            // doc: r2Publication.Metadata,
+            duration: r2Publication.Metadata.Duration &&
+                this.convertSecondToDaysHoursMinutes(r2Publication.Metadata.Duration),
+            nbOfTracks: r2Publication.Metadata.AdditionalJSON?.tracks as number,
```

Comment:
As mentioned in a previous code comment, I think `nbOfTracks` and `duration` should be optional properties, so if I am not mistaken `as number` can be removed and the TypeScript compiler will accept `AdditionalJSON?.tracks` as being potentially `undefined` (note: not `null` ... but this doesn't matter right now because Thorium does not have `strictNullChecks` in its `tsconfig.js` ;)

Reply:
https://github.com/readium/readium-desktop/pull/973/commits/b38967b14f3898ee4788b5a41a2392815fe9294e

Reply:
Oh, my bad, I forgot that `AdditionalJSON` is `AnyJson`, so naturally `as number` is needed! Thanks.